src/main/resources/lib/hudson/test/aggregated-failed-tests.jelly

```diff
@@ -52,7 +52,7 @@ THE SOFTWARE.
           </tr>
           <j:forEach var="f" items="${report.result.failedTests}" varStatus="i">
             <tr>
-              <td class="pane no-wrap"><t:failed-test result="${f}" url="../${report.child.project.shortUrl}testReport/${f.getRelativePathFrom(report.result)}"/></td>
+              <td class="pane no-wrap"><t:non-passed-test result="${f}" url="../${report.child.project.shortUrl}testReport/${f.getRelativePathFrom(report.result)}"/></td>
```

Comment:
Ditto, for renaming a reusable tag. There appears to be at least one[1] external use of this.

[1] https://github.com/jenkinsci/mentor-questa-vrm-plugin/blob/04079b04117d351878e1be16d19c79fa38ed2568/src/main/resources/com/mentor/questa/vrm/jenkins/QuestaVrmRegressionResult/body.jelly#L45